changes/unreleased/Added-20250722-093147.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Added
+body: 'A new environmental variable CACHE_ENABLED has been added. By default it is set to true, which enables cache in Vertica Operator. '
```

Comment:
This should be about the general new feature which is adding a cache. We don't need to mention the new env var.